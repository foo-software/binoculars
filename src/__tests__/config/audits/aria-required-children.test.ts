import auditAriaRequiredChildren from '../../../config/audits/aria-required-children';

describe('aria-required-children audit', () => {
  it('should match snapshot', () => {
    expect(auditAriaRequiredChildren('en')).toMatchSnapshot();
  });
});
