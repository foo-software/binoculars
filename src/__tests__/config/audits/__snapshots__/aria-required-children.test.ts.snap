// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`aria-required-children audit should match snapshot 1`] = `
Object {
  "description": "Some ARIA parent roles must contain specific child roles to perform their intended accessibility functions. [Learn more](https://web.dev/aria-required-children/).",
  "title": "Elements with an ARIA \`[role]\` that require children to contain a specific \`[role]\` have all required children.",
}
`;
